src/pages/components/content-switcher/style.mdx

```diff
@@ -68,70 +128,90 @@ Content switcher label text should be set in sentence case, with only the first
 word in a phrase and any proper nouns capitalized. The label text should not
 exceed three words.
 
-| Element | Font-size (px/rem) | Font-weight   | Type token         |
-| ------- | ------------------ | ------------- | ------------------ |
-| Label   | 14 / 0.875         | Regular / 400 | `$body-compact-01` |
+| Element    | Font size (px/rem) | Font weight   | Type token         |
```

Comment:
Just a small adjustment that I recently learned from @laurenmrice:
(Could you please fix the spacing of this column for all the rows below before committing the change?)

```suggestion
| Element    | px / rem  | Font weight   | Type token         |
```

Reply:
Thanks, Kritvi. I'd keep this for now. Most of our docs have this style for the Typography section. They also have a hyphen in between. Maybe we can talk more about this for the style template update

![image](https://github.com/carbon-design-system/carbon-website/assets/77697351/c8da3b4c-8bb1-474f-b4f9-4e3b87824443)

Reply:
> Thanks, Kritvi. I'd keep this for now. Most of our docs have this style for the Typography section. They also have a hyphen in between. Maybe we can talk more about this for the style template update
> 
> ![image](https://private-user-images.githubusercontent.com/77697351/316015965-c8da3b4c-8bb1-474f-b4f9-4e3b87824443.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEzNDc5NzcsIm5iZiI6MTcxMTM0NzY3NywicGF0aCI6Ii83NzY5NzM1MS8zMTYwMTU5NjUtYzhkYTNiNGMtOGJiMS00NzRmLWI0ZjktNGUzYjg3ODI0NDQzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI1VDA2MjExN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYxYmY4MTVmNWQwNWU3M2U2NGI2ODQzYjU2ZTg1NDExMGFlMWNhNDcxNjQ3MDM1ZjVmYzM5M2M4MzE2ZDQ4ZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.h1q7bHyEJZZuXjENC6XYwk_MHcR4-Fan2iREW04V1FE)


Anytime Thy!!
Yes, honestly, I also thought the same. However, Lauren pointed this out during one of my doc update reviews, so I learned from that. But yes, we can definitely discuss it further.